js/interface.js

```diff
@@ -1929,7 +1928,6 @@ var DynamicLists = (function() {
       $filterAccordionContainer.append($newPanel);
 
       if (data.logic === 'empty' || data.logic === 'notempty') {
-        $newPanel.find('.panel-title-text .value, #value-dash, #filter-value-type').addClass('hidden');
```

Comment:
The `#filter-value-type` UI is not relevant when data.log is `empty` or `notempty`. This means we can't remove this line of code.

Reply:
@tonytlwu, fixed